package.json

```diff
@@ -1,31 +1,30 @@
 {
-  "name": "hsfp.toyko",
+  "name": "hsfp.tokyo",
   "version": "0.1.0",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "test": "jest"
+    "lint": "next lint"
   },
   "dependencies": {
-    "@tailwindcss/forms": "^0.3.3",
-    "@testing-library/react": "^11.2.6",
-    "@types/jest": "^26.0.23",
-    "classnames": "^2.3.1",
-    "jest": "^26.6.3",
-    "next": "10.2.0",
-    "next-i18next": "^8.2.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "ts-jest": "^26.5.5"
+    "@types/jest": "^29.5.5",
```

Comment:
Not sure why I originally had jest as a dependency instead of a dev dependency, but we should be able to move over all jest related deps to dev.